pkg/rbd/rbd_journal.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 
 	"github.com/ceph/ceph-csi/pkg/util"
+	"golang.org/x/net/context"
```

Comment:
(nit) Should we just import "context"? Currently we have a mix as it seems and documentation for the package in line 23 states:
"As of Go 1.7 this package is available in the standard library under the name context. https://golang.org/pkg/context. "